src/components/table/LightTable.tsx

```diff
@@ -289,6 +293,8 @@ function LightTable({
     0,
   );
   const effectiveTableWidth = _.isNaN(tableWidth) || !supportsResize ? '100%' : tableWidth;
+  const showPagination = (rows && rows.length > 0) || infinitePaging;
```

Comment:
Do we want to show it if `rows.length === 0` and `infinitePaging`?  I would think that we'd want to hide in that case

Reply:
Yes! I had it originally hiding the pagination if there were no results, and we were infinite paging, but the problem was that if you paged to no results, you couldnt page back. You would be forced to hard refresh the page to continue seeing data